### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#9] [Announcements]
# about

an AI researcher going back to school for immunology

https://rachel.fast.ai/about.html

**Comment by aleenashabbir** [comment]
Really loved your TedX talk!

**Reply by racheltho** [reply]
Thank you!

**Comment by mimmiv** [comment]
Hello Rachel,

I recently read "An Introduction to Deep Learning for Tabular Data", where you reference the Lesson 3 Jupyter notebook for example code. However, the link appears to be broken, leading to a 404 error.

I’m currently working on handling categorical features in unlabeled tabular data, with the goal of applying a Self-Organizing Map (SOM) after preprocessing. Do you have any suggestions for example code or articles that could help with this task?

I’d really appreciate any guidance you can provide.

**Reply by racheltho** [reply]
Chapter 9 of the Fastai book covers Tabular Data in much more detail.  It is available for free here and works through an example in detail: https://github.com/fastai/fastbook/blob/master/09_tabular.ipynb

**Comment by mv2731** [comment]
Do you ever mentor younger women scientists? I work in bioML and will be pursuing a computer science PhD, but I am very interested to hear about your pivot into a second, more biologically-focused PhD and what you've learned throughout your career :)
